### ISSUE TITLE
Windows binary throwing error: '...need to compile the c_overviewer module...'

### ISSUE THREAD

**Issue by KariTrace** [overviewer/Minecraft-Overviewer#407]
\o great job so far guys. Overviewer is look just awesome!

I way back when I found Overviewer I got v0.0.6 up and running on my server. Fast forwards awhile and I see you guys have done a bunch of improvement and would liek to update to the recent version. I've tried the win86_32-release-0.1.4.zip binary as well as win86_32-release-0.1.4-33-gd2c9b80.zip and win86_32-release-0.1.4-59-g9cdfe0e.zip in a Windows XP SP2 box.

Steps: downloaded, extracted, attempted to run via the command: 'overviewer.exe' and w/ the --help.

tl;dr: I keep getting the "You need the compile the c_overviewer module to run Minecraft Overviewer...". Are not the binaries suppose to be pre-compiled? What am I missing here?

(Side note: I did install the 2008 C++ redist package from MS.)
